### ISSUE TITLE
oguzctnoglu

### ISSUE THREAD

**Issue by oguzctnoglu** [oguzctnoglu/oguzctnoglu#1]
<div align="center"> <img src="https://capsule-render.vercel.app/api?type=waving&color=gradient&customColorList=6,11,20&height=220&section=header&text=Merhaba,%20Ben%20Örnek%20Kullanıcı%20👋&fontSize=42&fontColor=ffffff&animation=fadeIn&fontAlignY=35&desc=Computer%20Engineering%20Student%20%7C%20AI%20%2F%20ML%20Enthusiast&descAlignY=55&descSize=18" width="100%"/> <a href="https://git.io/typing-svg"> <img src="https://readme-typing-svg.demolab.com?font=Fira+Code&weight=600&size=24&duration=3000&pause=800&color=38BDF8&center=true&vCenter=true&width=650&lines=4.+S%C4%B1n%C4%B1f+Bilgisayar+M%C3%BChendisli%C4%9Fi+%C3%96%C4%9Frencisiyim;.NET+Core+%26+Spring+Boot+ile+%C3%A7al%C4%B1%C5%9F%C4%B1yorum;AI+%2F+ML+Alan%C4%B1na+Tutkuyla+Ba%C4%9Fl%C4%B1y%C4%B1m+%F0%9F%9A%80;Medium%27da+Bilgi+Payla%C5%9F%C4%B1yorum+%E2%9C%8D%EF%B8%8F" alt="Typing SVG" /> </a> <br> <img src="https://komarev.com/ghpvc/?username=ornek-kullanici&label=Profile%20Views&color=38BDF8&style=for-the-badge" /> <img src="https://img.shields.io/badge/Status-Learning%20%26%20Building-38BDF8?style=for-the-badge" /> </div> <br> 
isim:       Örnek Kullanıcı
rol:        Bilgisayar Mühendisi
odak:       .NET Core · Spring Boot · Yapay Zeka / Makine Öğrenmesi
şu_an:      Yeni teknolojiler öğreniyor ve projeler geliştiriyor
ilgi_alanı: Backend Geliştirme, AI/ML, Yazılım Mimarisi
🎓 3,51 not ortalaması ile Bilgisayar Mühendisliğinden mezun oldum.
💻 .NET Core ve Spring Boot ile backend geliştirme yapıyorum
🤖 AI / ML alanına derin bir ilgi duyuyorum ve bu alanda kendimi geliştiriyorum
🌱 Her gün yeni bir şeyler öğrenmeye devam ediyorum
📫 Bana ulaşmak veya iş birliği yapmak isterseniz aşağıdaki kanallardan iletişime geçebilirsiniz
<br> <div align="center"> <a href="mailto:ornek.kullanici@example.com"> <img src="https://img.shields.io/badge/Gmail-EA4335?style=for-the-badge&logo=gmail&logoColor=white" /> </a> <a href="https://linkedin.com/in/ornek-kullanici"> <img src="https://img.shields.io/badge/LinkedIn-0A66C2?style=for-the-badge&logo=linkedin&logoColor=white" /> </a> <a href="https://medium.com/@ornek-kullanici"> <img src="https://img.shields.io/badge/Medium-000000?style=for-the-badge&logo=medium&logoColor=white" /> </a> <a href="https://www.hackerrank.com/ornek-kullanici"> <img src="https://img.shields.io/badge/HackerRank-2EC866?style=for-the-badge&logo=hackerrank&logoColor=white" /> </a> <a href="https://leetcode.com/ornek-kullanici/"> <img src="https://img.shields.io/badge/LeetCode-FFA116?style=for-the-badge&logo=leetcode&logoColor=black" /> </a> </div> <br> <div align="center"> 
Diller & Frameworkler
<img src="https://skillicons.dev/icons?i=java,cs,python,dotnet,spring,html,css,js&theme=dark" /> <br> 
Veritabanı, Araçlar & AI/ML
<img src="https://skillicons.dev/icons?i=postgresql,docker,git,github,vscode,postman,pytorch,tensorflow&theme=dark" /> </div> <br> <div align="center"> <img height="165" src="https://github-readme-streak-stats.herokuapp.com/?user=ornek-kullanici&theme=tokyonight&hide_border=true&background=0D1117&stroke=38BDF8&ring=38BDF8&fire=38BDF8&currStreakLabel=38BDF8" /> <br> <div align="center"> <img src="https://raw.githubusercontent.com/platane/platane/output/github-contribution-grid-snake.svg" alt="snake animation" width="100%"/> </div> <div align="center"> <img src="https://capsule-render.vercel.app/api?type=waving&color=gradient&customColorList=6,11,20&height=150&section=footer" width="100%"/> 
⭐ Profilime uğradığınız için teşekkürler! Birlikte kod yazmak dileğiyle 🚀
</div>
